consensus-types/blocks/execution.go

```diff
@@ -155,6 +155,16 @@ func (e executionPayload) WithdrawalsRoot() ([]byte, error) {
 	return nil, ErrUnsupportedGetter
 }
 
+// PbV1 --
+func (e executionPayload) PbV1() (*enginev1.ExecutionPayload, error) {
```

Comment:
Do we really want to name things `PbV1`/`PbV2` instead of `Pb`/`PbCapella` or `PbBellatrix`/`PbCapella`? It might be misleading because all protobufs are under the `enginev1` package.